.prettierrc.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 module.exports = {
+  arrowParens: 'avoid',
```

Comment:
IMHO, we should not override this. The default new Ember app blueprint will not include this customization (based on https://github.com/emberjs/rfcs/blob/master/text/0628-prettier.md), and we shouldn't arbitrarily differ from that.